Altis_Life.Altis/core/functions/fn_keyHandler.sqf

```diff
@@ -6,28 +6,31 @@
 *    Description:
 *    Main key handler for event 'keyDown'.
 */
-private ["_handled","_shift","_alt","_code","_ctrl","_alt","_ctrlKey","_veh","_locked","_interactionKey","_mapKey","_interruptionKeys"];
-_ctrl = _this select 0;
-_code = _this select 1;
-_shift = _this select 2;
-_ctrlKey = _this select 3;
-_alt = _this select 4;
-_speed = speed cursorObject;
-_handled = false;
+params [
+    ["_ctrl", displayNull, [displayNull]],
+    ["_code", 0, [0]],
+    ["_shift", false, [false]],
+    ["_ctrlKey", false, [false]],
+    ["_alt", false, [false]]
+];
 
-_interactionKey = if (count (actionKeys "User10") isEqualTo 0) then {219} else {(actionKeys "User10") select 0};
-_mapKey = (actionKeys "ShowMap" select 0);
-//hint str _code;
-_interruptionKeys = [17,30,31,32]; //A,S,W,D
+private _speed = speed cursorObject;
+private _handled = false;
+private _veh = objNull;
+private _locked = false;
+
```

Comment:
`_veh` & `_locked` shouldn't need to be declared in a global scope, looking at the flow of code they will always be assigned to before being used.